User.py

```diff
@@ -15,6 +15,9 @@ def __init__(self, unique_id: str) -> None:
             if self.access.admin_status != 1:
                 # and if user isn't admin
                 self.clock = Clock(unique_id)
+            elif self.access.admin_status == 1:
```

Comment:
Couldn't you just have an else statement instead of checking for an admin status? Since you check if the user is admin above, but if its not admin the only other option would be admin, right?

Reply:
I just leaved it this way to explicitly check if the number for admin_status is 1. I did it this way so that any number other than 1 wouldn't be an admin. However I will change the first conditional to something like if number is 0 to make sure that only the number 1 and 0 are used.